nsv13/code/modules/overmap/fighters/fighters.dm

```diff
@@ -29,8 +29,8 @@ After going through this checklist, you're ready to go!
 */
 
 /obj/structure/overmap/fighter
-	name = "Viper"
-	desc = "A space faring fighter craft."
+	name = "Fighter - PARENT"
+	desc = "THIS IS A PARENT STRUCTURE AND SHOULD NOT BE SPAWNED"

```

Comment:
Can't wait to see one of these on the map lol

Reply:
Can't wait to see all the parent objects where they should't be.